src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1342,14 +1343,17 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         $classToServices = [
             LocoProviderFactory::class => 'translation.provider_factory.loco',
+            PoEditorProviderFactory::class => 'translation.provider_factory.poeditor',
         ];
 
         $parentPackages = ['symfony/framework-bundle', 'symfony/translation', 'symfony/http-client'];
 
         foreach ($classToServices as $class => $service) {
-            $package = sprintf('symfony/%s-translation-provider', substr($service, \strlen('translation.provider_factory.')));
+            switch ($package = substr($service, \strlen('translation.provider_factory.'))) {
+                case 'poeditor': $package = 'po-editor'; break;
```

Comment:
I'd be in favor or naming the package poeditor and skipping this line

Reply:
It was the original name, but @fabpot commented about it https://github.com/symfony/symfony/pull/38475#discussion_r588856406

Reply:
My comment was related to the package name. The package name should be directly derived from the bridge name. So, if the bridge name is `PoEditor`, the package name must be `po-editor`.

Reply:
I agree, let me know if you prefer to name the bridge `PoEditor` or `Poeditor` (note that the *real* name of the company is POEditor), and I will adapt the package name accordingly.